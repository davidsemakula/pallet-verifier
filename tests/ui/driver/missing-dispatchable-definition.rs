use std::marker::PhantomData;

pub trait Config {}

pub enum Call<T: Config> {
    __Ignore(PhantomData<T>),
    do_something { val: u8 },
}

pub struct Pallet<T: Config>(PhantomData<T>);
