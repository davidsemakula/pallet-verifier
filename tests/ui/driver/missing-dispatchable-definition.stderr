warning: Couldn't find definition for dispatchable: `do_something`
   |
   = note: This is most likely a bug in pallet-verifier.
   = help: Please consider filling a bug report at https://github.com/davidsemakula/pallet-verifier/issues

error: Failed to find definitions for any dispatchable function in public interface
   |
   = note: pallet-verifier can only analyze FRAME dispatchable functions
   = help: Learn more at https://github.com/davidsemakula/pallet-verifier

error: aborting due to 1 previous error; 1 warning emitted

